src/plugins/json.js

```diff
@@ -24,8 +24,8 @@ function init(_self, _opts) {
 
   const parser = new JSONParser(this.options);
 
-  parser.on('data', ({ key, value }) => {
-    this.emit('field', key, value);
+  parser.on('data', (fields) => {
+    this.fields = fields;
```

Comment:
@GrosSacASac 

it should still emit `field` event

Reply:
it was done https://github.com/node-formidable/formidable/issues/760
if the json is just a number, or string it can't emit field with key value